src/mds/Server.cc

```diff
@@ -6078,6 +6078,9 @@ void Server::handle_set_vxattr(MDRequestRef& mdr, CInode *cur)
 
     bool val;
     try {
+      if (value.length() == 0) {
```

Comment:
For readability, put at the start of this method:

`bool is_rmxattr = (value.length() == 0);`

Then use that.

Reply:
I ended up using `bool is_rmxattr = (req->get_op() == CEPH_MDS_OP_RMXATTR);` instead. That way seemed more straight forward.